app/src/main/java/com/infomaniak/drive/ui/BasePreviewSliderFragment.kt

```diff
@@ -73,6 +72,7 @@ abstract class BasePreviewSliderFragment : Fragment(), FileInfoActionsView.OnIte
     override val currentContext by lazy { requireContext() }
     override lateinit var currentFile: File
 
+    val drivePermissions: DrivePermissions = DrivePermissions()
```

Comment:
I can't as it would cause the compilator to crash. 
This is because its the children file also implements an interface that require this drivePermissions.